cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/schedule/AbstractSchedulerService.java

```diff
@@ -229,6 +233,26 @@ public ScheduleState scheduleState(ProgramId program, SchedulableProgramType pro
     }
   }
 
+  @Nullable
+  public static String removeAppVersion(String scheduleId) {
```

Comment:
Especially this is an abstract base class, it's really out of place to host util method.